packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1595,6 +1677,30 @@ class EditableText extends StatefulWidget {
   /// {@macro flutter.services.TextInputConfiguration.enableIMEPersonalizedLearning}
   final bool enableIMEPersonalizedLearning;
 
+  /// {@template flutter.widgets.editableText.contentInsertionConfiguration}
+  /// Configuration of handler for media content inserted via the system input
+  /// method.
+  ///
+  /// By default, no handler will be created for inserted content. Set
```

Comment:
maybe: Defaults to null in which case software keyboard rich content insertion will be disabled, and the system may display a message informing the user ....